spec.html

```diff
@@ -449,6 +449,7 @@ <h1>Organization of This Specification</h1>
     <p>Clauses 6-9 define the execution environment within which ECMAScript programs operate.</p>
     <p>Clauses 10-16 define the actual ECMAScript programming language including its syntactic encoding and the execution semantics of all language features.</p>
     <p>Clauses 17-26 define the ECMAScript standard library. They include the definitions of all of the standard objects that are available for use by ECMAScript programs as they execute.</p>
+    <p>Clause 27 describes the memory consistency model of SharedArrayBuffer objects and methods of the Atomics object.</p>
```

Comment:
How about "the memory consistency model of accesses on SharedArrayBuffer-backed memory"? The model doesn't talk about the objects themselves.

Reply:
Fixed.